patches/AdjustDocTitle.java

```diff
@@ -0,0 +1,36 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS org.yaml:snakeyaml:2.0
+
+import org.yaml.snakeyaml.DumperOptions;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.util.Collections;
+import java.util.Map;
+
+public class AdjustDocTitle {
+
+    public static void main(String... args) throws Exception {
+        File antoraFile = new File("docs/antora.yml");
+        if (antoraFile.exists()) {
+            final DumperOptions options = new DumperOptions();
+            options.setIndent(3);
+            options.setIndicatorIndent(2);
+            options.setDefaultFlowStyle(DumperOptions.FlowStyle.BLOCK);
+            Yaml yaml = new Yaml(options);
+            Map<String, String> data = Collections.emptyMap();
+            try (var is = new FileInputStream(antoraFile)) {
+                data = yaml.load(is);
+            }
+            String title = data.get("title");
+            if (title.startsWith("Quarkus ")) {
```

Comment:
You need to handle `Quarkus - Java Embedded Framework` first.

And could we also drop the ` Extensions for Quarkus` from `CXF Extensions for Quarkus`?

Reply:
Because they are specific, it's easier to create those PRs manually for this case